src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -23,10 +24,11 @@ export interface TransactionDetailsSummaryItemProps
   useLastSubmittedOffer?: boolean
   offerContextPrice?: "LIST_PRICE" | "LAST_OFFER"
   showOfferNote?: boolean
-  placeholderOverride?: string | null
+  transactionStep?: string | null
```

Comment:
I replaced 'placeholderOverride' with 'transactionStep' so that we can conditionally change the transaction details summary more easily.

Reply:
Makes sense to me!